summariser/src/lib.rs

```diff
@@ -21,7 +21,6 @@ pub fn execute_report_for_run_summary(
     summary: RunSummary,
 ) -> Option<BoxFuture<'static, anyhow::Result<SummaryOutput>>> {
     let name = &summary.scenario_name;
-
```

Comment:
The whitespace is meaningful here, to separate statements. The clone is kept with the `match` because it's done to work with the `move` blocks.

Reply:
Okay. That must have gotten accidentally removed when I removed a line of tracing for debugging. Reinserted - https://github.com/holochain/wind-tunnel/pull/318/commits/b445f87438497a90f76710efd69d198b880a94dd